code/modules/flufftext/Hallucination.dm

```diff
@@ -11,11 +11,16 @@ Gunshots/explosions/opening doors/less rare audio (done)
 
 */
 
+#define SCREWYHUD_NONE 0
+#define SCREWYHUD_CRIT 1
+#define SCREWYHUD_DEAD 2
+#define SCREWYHUD_HEALTHY 3
```

Comment:
i couldn't find anywhere else in the code where they were referenced, adn i implemented a more versatile hallucination for alerts anyway